documentation/guides/deployment.md

```diff
@@ -0,0 +1,220 @@
+# Deployments
+
+This document covers API and Frontend deployments. It does not include
+information regarding the ingestion server or our Airflow instance.
+
+## Real-time Deployment Information
+
+### Frontend
+
+| Environment | Deployment Log                                                                                        | URL                                | Version Endpoint                                                 |
+| ----------- | ----------------------------------------------------------------------------------------------------- | ---------------------------------- | ---------------------------------------------------------------- |
+| Staging     | [Deployment Log](https://github.com/WordPress/openverse/actions/workflows/deploy-staging-nuxt.yml)    | <https://staging.openverse.org>    | [`/version.json`](https://staging.openverse.org/version.json)    |
+| Production  | [Deployment Log](https://github.com/WordPress/openverse/actions/workflows/deploy-production-nuxt.yml) | <https://production.openverse.org> | [`/version.json`](https://production.openverse.org/version.json) |
```

Comment:
Is the `production.` necessary here?

Reply:
It isn't, `openverse.org` is just a CNAME rule for `production.openverse.org` and they resolve to the same service, but I find being explicit about it is helpful.